src/grapple.c

```diff
@@ -429,7 +430,10 @@ void GrappleThrow()
 	}
 
 	float hasteMultiplier =	(cvar("k_ctf_rune_power_hst") / 16) + 1;
-	float throwSpeed = cvar("k_ctf_hookstyle") != 3 ? CR_THROW_SPEED : THROW_SPEED;	
+	float throwSpeed = cvar("k_ctf_hookstyle") != 3 ? NEW_THROW_SPEED : THROW_SPEED;	
+	
+	if (cvar("k_ctf_hookstyle") == 4) throwSpeed = CR_THROW_SPEED;
+
```

Comment:
might want to convert these to a series of `if` statements, keeping the ternary doesn't make a ton of sense anymore.